README.md
#<b> IMPORTANTE  </b>


<p> Para crear usuarios y la base de datos usar (" php artisan db:seed --class=DatabaseSeeder "), el admin se crea como admin@admin.com y su contraseña es "admin"
los usuarios normales llevan por defecto la contraseña "password".
</p>

<p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400" alt="Laravel Logo"></a></p>

<p align="center">
<a href="https://travis-ci.org/laravel/framework"><img src="https://travis-ci.org/laravel/framework.svg" alt="Build Status"></a>
<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/dt/laravel/framework" alt="Total Downloads"></a>
<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/v/laravel/framework" alt="Latest Stable Version"></a>
<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
</p>

## About Laravel

Laravel is a web application framework with expressive, elegant syntax. We believe development must be an enjoyable and creative experience to be truly fulfilling. Laravel takes the pain out of development by easing common tasks used in many web projects, such as:

- [Simple, fast routing engine](https://laravel.com/docs/routing).
- [Powerful dependency injection container](https://laravel.com/docs/container).
- Multiple back-ends for [session](https://laravel.com/docs/session) and [cache](https://laravel.com/docs/cache) storage.
- Expressive, intuitive [database ORM](https://laravel.com/docs/eloquent).
- Database agnostic [schema migrations](https://laravel.com/docs/migrations).
- [Robust background job processing](https://laravel.com/docs/queues).
- [Real-time event broadcasting](https://laravel.com/docs/broadcasting).

Laravel is accessible, powerful, and provides tools required for large, robust applications.

## Learning Laravel

Laravel has the most extensive and thorough [documentation](https://laravel.com/docs) and video tutorial library of all modern web application frameworks, making it a breeze to get started with the framework.

You may also try the [Laravel Bootcamp](https://bootcamp.laravel.com), where you will be guided through building a modern Laravel application from scratch.

If you don't feel like reading, [Laracasts](https://laracasts.com) can help. Laracasts contains over 2000 video tutorials on a range of topics including Laravel, modern PHP, unit testing, and JavaScript. Boost your skills by digging into our comprehensive video library.

## Laravel Sponsors

We would like to extend our thanks to the following sponsors for funding Laravel development. If you are interested in becoming a sponsor, please visit the Laravel [Patreon page](https://patreon.com/taylorotwell).

### Premium Partners

- **[Vehikl](https://vehikl.com/)**
- **[Tighten Co.](https://tighten.co)**
- **[Kirschbaum Development Group](https://kirschbaumdevelopment.com)**
- **[64 Robots](https://64robots.com)**
- **[Cubet Techno Labs](https://cubettech.com)**
- **[Cyber-Duck](https://cyber-duck.co.uk)**
- **[Many](https://www.many.co.uk)**
- **[Webdock, Fast VPS Hosting](https://www.webdock.io/en)**
- **[DevSquad](https://devsquad.com)**
- **[Curotec](https://www.curotec.com/services/technologies/laravel/)**
- **[OP.GG](https://op.gg)**
- **[WebReinvent](https://webreinvent.com/?utm_source=laravel&utm_medium=github&utm_campaign=patreon-sponsors)**
- **[Lendio](https://lendio.com)**

## Contributing

Thank you for considering contributing to the Laravel framework! The contribution guide can be found in the [Laravel documentation](https://laravel.com/docs/contributions).

## Code of Conduct

In order to ensure that the Laravel community is welcoming to all, please review and abide by the [Code of Conduct](https://laravel.com/docs/contributions#code-of-conduct).

## Security Vulnerabilities

If you discover a security vulnerability within Laravel, please send an e-mail to Taylor Otwell via [taylor@laravel.com](mailto:taylor@laravel.com). All security vulnerabilities will be promptly addressed.

## License

The Laravel framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).


CHANGELOG.md
# Release Notes

## [Unreleased](https://github.com/laravel/laravel/compare/TIP...master)

## [TIP](https://github.com/laravel/laravel/compare/v9.3.8...TIP) - 2022-10-20

Para crear datos usar ("php artisan db:seed --class=DatabaseSeeder")

## [v9.3.8](https://github.com/laravel/laravel/compare/v9.3.7...v9.3.8) - 2022-09-20

### Changed

- Validation added `required_if_accepted` by @luisprmat in https://github.com/laravel/laravel/pull/5987

## [v9.3.7](https://github.com/laravel/laravel/compare/v9.3.6...v9.3.7) - 2022-09-02

### Changed

- Make email unique by @martin-ro in https://github.com/laravel/laravel/pull/5978

## [v9.3.6](https://github.com/laravel/laravel/compare/v9.3.5...v9.3.6) - 2022-08-29

### Changed

- Bump Vite plugin version by @timacdonald in https://github.com/laravel/laravel/pull/5977

## [v9.3.5](https://github.com/laravel/laravel/compare/v9.3.4...v9.3.5) - 2022-08-22

### Changed

- `max_digits` and `min_digits` validation translations by @danharrin in https://github.com/laravel/laravel/pull/5975
- Use short closure by @taylorotwell in https://github.com/laravel/laravel/commit/7b17f5f32623c2ee75f2bff57a42bb8f180ac779
- Use except by @taylorotwell in https://github.com/laravel/laravel/commit/e2e25f607a894427d6545f611ad3c8d94d022e9d

## [v9.3.4](https://github.com/laravel/laravel/compare/v9.3.3...v9.3.4) - 2022-08-15

### Changed

- Add ValidateSignature middleware for ignore params by @valorin in https://github.com/laravel/laravel/pull/5942

## [v9.3.3](https://github.com/laravel/laravel/compare/v9.3.2...v9.3.3) - 2022-08-03

### Changed

- Validation added `doesnt_end_with` translation by @kichetof in https://github.com/laravel/laravel/pull/5962

## [v9.3.2](https://github.com/laravel/laravel/compare/v9.3.1...v9.3.2) - 2022-08-01

### Changed

- Update Sanctum by @suyar in https://github.com/laravel/laravel/pull/5957
- Allow Pest plugin in Composer by @driesvints in https://github.com/laravel/laravel/pull/5959

## [v9.3.1](https://github.com/laravel/laravel/compare/v9.3.0...v9.3.1) - 2022-07-26

### Changed

- Update font delivery by @abenerd in https://github.com/laravel/laravel/pull/5952
- Don't need to ignore vite config file by @GrahamCampbell in https://github.com/laravel/laravel/pull/5953

## [v9.3.0](https://github.com/laravel/laravel/compare/v9.2.1...v9.3.0) - 2022-07-20

### Added

- Uses `laravel/pint` for styling by @nunomaduro in https://github.com/laravel/laravel/pull/5945

### Changed

- Bump axios version by @ankurk91 in https://github.com/laravel/laravel/pull/5946
- Vite 3 support by @timacdonald in https://github.com/laravel/laravel/pull/5944

## [v9.2.1](https://github.com/laravel/laravel/compare/v9.2.0...v9.2.1) - 2022-07-13

### Changed

- Add auth.json to skeleton by @driesvints in https://github.com/laravel/laravel/pull/5924
- Update `bootstrap.js` by @irsyadadl in https://github.com/laravel/laravel/pull/5929
- Add default reloading to skeleton by @timacdonald in https://github.com/laravel/laravel/pull/5927
- Update to the latest version of laravel-vite-plugin by @jessarcher in https://github.com/laravel/laravel/pull/5943

## [v9.2.0](https://github.com/laravel/laravel/compare/v9.1.10...v9.2.0) - 2022-06-28

### Added

- Vite by @jessarcher in https://github.com/laravel/laravel/pull/5904
- Added support for easy development configuration in bootstrap.js by @rennokki in https://github.com/laravel/laravel/pull/5900

### Changed

- Sorted entries in the `en` validation translations file by @FaridAghili in https://github.com/laravel/laravel/pull/5899

## [v9.1.10](https://github.com/laravel/laravel/compare/v9.1.9...v9.1.10) - 2022-06-07

### Changed

- Add language line by @taylorotwell in https://github.com/laravel/laravel/commit/b084aacc5ad105e39c2b058e9523e73655be8d1f
- Improve Pusher configuration for easy development by @oanhnn in https://github.com/laravel/laravel/pull/5897

## [v9.1.9](https://github.com/laravel/laravel/compare/v9.1.8...v9.1.9) - 2022-05-28

### Changed

- Switch to ESM imports by @jessarcher in https://github.com/laravel/laravel/pull/5895

## [v9.1.8](https://github.com/laravel/laravel/compare/v9.1.7...v9.1.8) - 2022-05-05

### Changed

- Add local_domain option to smtp configuration by @bintzandt in https://github.com/laravel/laravel/pull/5877
- Add specific test user in seeder by @driesvints in https://github.com/laravel/laravel/pull/5879

## [v9.1.7](https://github.com/laravel/laravel/compare/v9.1.6...v9.1.7) - 2022-05-03

### Changed

- Deprecation log stack trace option by @driesvints in https://github.com/laravel/laravel/pull/5874

## [v9.1.6](https://github.com/laravel/laravel/compare/v9.1.5...v9.1.6) - 2022-04-20

### Changed

- Move password lines into main translation file by @taylorotwell in https://github.com/laravel/laravel/commit/db0d052ece1c17c506633f4c9f5604b65e1cc3a4
- Add missing maintenance to config by @ibrunotome in https://github.com/laravel/laravel/pull/5868

## [v9.1.5](https://github.com/laravel/laravel/compare/v9.1.4...v9.1.5) - 2022-04-12

### Changed

- Rearrange route methods by @osbre in https://github.com/laravel/laravel/pull/5862
- Add levels to handler by @taylorotwell in https://github.com/laravel/laravel/commit/a507e1424339633ce423729ec0ac49b99f0e57d7

## [v9.1.4](https://github.com/laravel/laravel/compare/v9.1.3...v9.1.4) - 2022-03-29

### Changed

- Add encryption configuration by @taylorotwell in https://github.com/laravel/laravel/commit/f7b982ebdf7bd31eda9f05f901bd92ed32446156

## [v9.1.3](https://github.com/laravel/laravel/compare/v9.1.2...v9.1.3) - 2022-03-29

### Changed

- Add an example to the class aliases by @nshiro in https://github.com/laravel/laravel/pull/5846
- Add username in config to use with phpredis + ACL by @neoteknic in https://github.com/laravel/laravel/pull/5851
- Remove "password" from validation lang by @mnastalski in https://github.com/laravel/laravel/pull/5856

resources/views/welcome.blade.php
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Home</title>

    <!-- Fonts -->
    <link href="https://fonts.bunny.net/css2?family=Nunito:wght@400;600;700&display=swap" rel="stylesheet">

    <!--Styles -->

    <style>
        * {
            margin: 0;
            padding: 0;
            box-sizing: border-box;
        }

        body {
            font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
        }

        .container {
            padding: 60px 0;
            width: 90%;
            max-width: 1000px;
            margin: auto;
            overflow: hidden;
        }

        .titulo {
            color: #642a73;
            font-size: 30px;
            text-align: center;
            margin-bottom: 60px;
        }

        header {
            width: 100%;
            height: 600px;

            background: rgb(2, 0, 36);
            background: linear-gradient(90deg, rgba(2, 0, 36, 1) 0%, rgba(103, 9, 121, 0.6024159663865547) 60%, rgba(0, 212, 255, 0.6024159663865547) 100%), url(https://cursos.aiu.edu/images/Cuando-una-institucion-educativa-calla-ENTRADA-1.jpg);

            background-size: cover;
            background-attachment: fixed;
            position: relative;
        }

        nav {
            text-align: right;
            padding: 30px 50px 0 0;
        }

        nav>a {
            color: #fff;
            font-weight: 300;
            text-decoration: none;
            font-size: 16px;
            margin-right: 10px;
        }

        nav>a:hover {
            text-decoration: underline;
        }

        header .text-header {
            display: flex;
            height: 430px;
            width: 100%;
            align-items: center;
            justify-content: center;
            flex-direction: column;
            text-align: center;
        }

        .text-header h1 {
            font-size: 50px;
            color: #fff;
        }

        .text-header h2 {
            font-size: 30px;
            font-weight: 300;
            color: #fff;
        }

        .wave {
            position: absolute;
            bottom: 0;
            width: 100%;
        }

        .container-about-us {
            display: flex;
            justify-content: space-evenly;
        }

        .picture-about-us {
            width: 48%;
        }

        .container-about-us .contents-text {
            width: 48%;
        }

        .contents-text h3 {
            margin-bottom: 15px;
        }

        .contents-text h3 span {
            background: #4d0686;
            color: #fff;
            border-radius: 50%;
            display: inline-block;
            text-align: center;
            width: 30px;
            height: 30px;
            padding: 3px;
            box-shadow: 0 0 6px 0 rgba(0, 0, 0, .5);
            margin-right: 5px;
        }

        .contents-text p {
            padding: 0px 0px 30px 15px;
            font-weight: 300;
            text-align: justify;
        }

        .portfolio {
            background: #f2f2f2;
        }

        .gallery-port {
            display: flex;
            justify-content: space-evenly;
            flex-wrap: wrap;

        }

        .picture-port {
            width: 24%;
            margin-bottom: 10px;
            overflow: hidden;
            position: relative;
            cursor: pointer;
            box-shadow: 0 0 6px 0 rgba(0, 0, 0, .5);
        }

        .picture-port>img {
            width: 100%;
            height: 100%;
            object-fit: cover;
            display: block;
        }

        .hover-gallery {
            position: absolute;
            width: 100%;
            height: 100%;
            top: 0;
            transform: scale(0);
            background: hsla(273, 91%, 27%, 0.7);
            transition: transform .5s;
            display: flex;
            justify-content: center;
            align-items: center;
            flex-direction: column;
        }

        .hover-gallery img {
            width: 50px;
        }

        .hover-gallery p {
            color: #fff;
        }

        .picture-port:hover .hover-gallery {
            transform: scale(1);
        }

        .cards {
            display: flex;
            justify-content: space-evenly;
        }

        .cards .card {
            background: #4d0686;
            display: flex;
            width: 46%;
            height: 200px;
            align-items: center;
            justify-content: space-evenly;
            border-radius: 5px;
            box-shadow: 0 0 6px 0 rgba(0, 0, 0, .5);
        }

        .cards .card img {
            width: 100px;
            height: 100px;
            object-fit: cover;
            border: 3px solid #fff;
            border-radius: 50%;
            display: block;
        }

        .cards .card > .content-text-card{
            width: 60%;
            color: #fff;

        }

        .cards .card > .content-text-card p{
            font-weight: 300;
            padding-top: 5px;
        }
        
        .about-services{
            background: #f2f2f2;
            padding-bottom: 30px;
        }

        .service-cont{
            display: flex;
            justify-content: space-between;
            align-items: center;
        }

        .service-ind{
            width: 28%;
            text-align: center;
        }

        .service-ind img{
            width: 90%;
        }

        .service-ind h3{
            margin: 10px 0;
        }

        .service-ind p{
            font-weight: 300;
            text-align: justify;
        }

        footer{
            background: #414141;
            padding: 60px 0 30px 0;
            margin: auto;
            overflow: hidden;
        }

        .container-footer{
            display: flex;
            width: 90%;
            justify-content: space-evenly;
            margin: auto;
            padding-bottom: 50px;
            border-bottom: 1px solid gray;
        }

        .content-foot{
            text-align: center;
        }

        .content-foot h4{
            border-bottom: 3px solid purple;
            padding-bottom: 5px;
            margin-bottom: 10px;
        }

        .content-foot p{
            color: #ccc;

        }

        .titulo-final{
            text-align: center;
            font-size: 24px;
            margin: 20px 0 0 0;
            color: lightgrey;
        }
    </style>
</head>

<body class="antialiased">
    <header>
        <nav>
            @if (Route::has('login'))
            @auth
            <a href="{{ url('/home') }}" class="text-sm text-gray-700 dark:text-gray-500 underline">Home</a>
            @else
            <a href="{{ route('login') }}" class="text-sm text-gray-700 dark:text-gray-500 underline">Log in</a>
            @if (Route::has('register'))
            <a href="{{ route('register') }}" class="ml-4 text-sm text-gray-700 dark:text-gray-500 underline">Register</a>
            @endif
            @endauth
            @endif

        </nav>
        <section class="text-header">
            <h1>Create and manage your institution</h1>
            <h2>In an easy and fast way from a web site</h2>
        </section>
        <div class="wave" style="height: 150px; overflow: hidden;"><svg viewBox="0 0 500 150" preserveAspectRatio="none" style="height: 100%; width: 100%;">
                <path d="M0.00,49.99 C150.00,150.00 349.20,-49.99 500.00,49.99 L500.00,150.00 L0.00,150.00 Z" style="stroke: none; fill: #fff;"></path>
            </svg>
        </div>
    </header>
    <main>
        <section class="container about-us">
            <h2 class="titulo">What do we offer?</h2>
            <div class="container-about-us">
                <img src="https://www.umsa.edu.mx/childcontrol/Sistemas_control/Administrador/Imagenes_plan/1_bannerM2_2020_06_03_18-15-43.png" alt="" class="picture-about-us">
                <div class="contents-text">
                    <h3><span>1</span>The best tools</h3>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero ab quidem culpa a doloribus, dolor reprehenderit totam quia omnis libero? Similique iure libero odio ratione harum assumenda eius perspiciatis enim?</p>
                    <h3><span>2</span>The best services</h3>
                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero ab quidem culpa a doloribus, dolor reprehenderit totam quia omnis libero? Similique iure libero odio ratione harum assumenda eius perspiciatis enim?</p>
                </div>
            </div>
        </section>
        <section class="portfolio">
            <div class="container">
                <h2 class="titulo">Portfolio</h2>
                <div class="gallery-port">
                    <div class="picture-port">
                        <img src="https://cursodeadministraciondeempresas.com/wp-content/uploads/2020/08/1.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://elinsignia.com/wp-content/uploads/2019/07/integracion-1.jpg" alt="" width="800px" height="600px">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://conceptodefinicion.de/wp-content/uploads/2019/05/Administraci%C3%B3n-.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://uem.edu.mx/wp-content/uploads/2021/01/6909-scaled-1024x732.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://es.pimex.co/blog/wp-content/uploads/sites/2/2020/08/9.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://www.softwarelogisticaydistribucion.eu/wp-content/uploads/2021/07/administracion-de-procesos.png" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://valls.email/wp-content/uploads/2020/02/Colas-de-proyecto-Business-Central.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                    <div class="picture-port">
                        <img src="https://ingenieriasamat.es/wp-content/uploads/2019/02/AutoProcesos-Portada-01-640x480.jpg" alt="">
                        <div class="hover-gallery">
                            <img src="https://www.pngkey.com/png/full/120-1209359_iconmonstr-target-3-icon-1-01-target-white.png" alt="">
                            <p>Our Work</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section>
            <div class="customers container">
                <h2 class="titulo">Our customers</h2>
                <div class="cards">
                    <div class="card">
                        <img src="https://orientacion.universia.net.co/imgs2011/imagenes/servicio-a-2020_01_27_190137@2x.jpg" alt="">
                        <div class="content-text-card">
                            <h4>Name</h4>
                            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quisquam, eum.</p>
                        </div>
                    </div>
                    <div class="card">
                        <img src="https://f.hubspotusercontent00.net/hubfs/53/Tiposdeclientes.jpeg" alt="">
                        <div class="content-text-card">
                            <h4>Name</h4>
                            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quisquam, eum.</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <section>
            <div class="about-services">
                <div class="container">
                    <h2 class="titulo">Our services</h2>
                    <div class="service-cont">
                        <div class="service-ind">
                            <img src="https://www.businessprocessincubator.com/wp-content/uploads/2020/06/www.flokzu.comapi-vs-open-source-01-1-1-5ad32e70ddf27d8d9034b20aa6751cf79c7c9800.png" alt="">
                            <h3>Name</h3>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit, voluptatem?</p>
                        </div>
                        <div class="service-ind">
                            <img src="https://www3.ubu.es/ubucevblog/wp-content/uploads/2020/05/2.jpg" alt="">
                            <h3>Name</h3>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit, voluptatem?</p>
                        </div>
                        <div class="service-ind">
                            <img src="https://www3.ubu.es/ubucevblog/wp-content/uploads/2020/05/1.jpg" alt="">
                            <h3>Name</h3>
                            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit, voluptatem?</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>
    </main>
    <footer>
        <div class="container-footer">
            <div class="content-foot">
                <h4>Phone</h4>
                <p>+57(318)-567-1345</p>
            </div>
            <div class="content-foot">
                <h4>Email</h4>
                <p>example@example.com</p>
            </div>
            <div class="content-foot">
                <h4>Location</h4>
                <p>Barranquilla-Atlantico Calle 59 #74B-145</p>
            </div>
        </div>
        <h2 class="titulo-final">&copy; Xero Desing | Cristian Arrieta</h2>
    </footer>

</body>

</html>